packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx

```diff
@@ -122,4 +126,13 @@ export class Tabs {
       this.onChange.emit({ tabId: tabIndex });
     }
   }
+
+  private blurTabButton(tabIndex: number) {
+    const buttons = document.getElementsByClassName(
+      'plmg-tab-button'
+    ) as HTMLCollectionOf<HTMLButtonElement>;
+    if (document.activeElement === buttons.item(tabIndex)) {
```

Comment:
I feel like this logic assume there is only one Tabs on the page. What if they are multiples?

Reply:
Now restricted to component scope.